packages/compass-crud/src/components/bulk-update-dialog.tsx

```diff
@@ -293,17 +294,28 @@ export default function BulkUpdateDialog({
     }
   }, [isOpen, wasOpen, updateText]);
 
+  const modalTitleAndButtonText = useMemo(() => {
+    if (typeof count !== 'number') {
+      return 'Update documents';
+    }
+
+    if (count === 1) {
+      return `Update 1 document`;
+    }
+
+    return `Update ${count} documents`;
+  }, [count]);
+
+  const bulkUpdateUpdateId = useId('bulk-update-update');
```

Comment:
Passing a default id just returns this id, is that the behavior we want? Why do we need to use the hook then?

Reply:
well .. TIL, `useId` doesn't take the id prefix as argument and always used it wrong :)